### PR TITLE
Add Rethereum Mainnet

### PR DESCRIPTION
**Pull request by JABirchall**
- Create eip155-622277.json

### CODE REVIEW

**Comment by Memorti** [comment]
Hello

File: _data/icons/rethereum.json
Comment (Akaslove99) [inline _data/icons/rethereum.json:1]: #

**Review by Akaslove99 — APPROVED** [review]

**Comment by JABirchall** [comment]
This is now ready for merge.

File: _data/chains/eip155-622277.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "features": [{ "name": "EIP155" }, { "name": "EIP1559" }],
+  "infoURL": "https://www.rethereum.org/",
+  "shortName": "rth",
+  "chainId": 622277,
+  "networkId": 622277,
+  "icon": "rethereum",
+  "explorers": [{
+    "name": "rethereum",
+    "url": "https://explorer.rethereum.org/",
```
Comment (ligi) [inline _data/chains/eip155-622277.json:23]:
```suggestion
    "url": "https://explorer.rethereum.org",
```

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Comment by JABirchall** [comment]
Ran prettier and commited changes.

**Review by ligi — APPROVED** [review]

**Comment by Ayid7676** [comment]
0xFbB638237383ac291251cea4974d8f60642e8A49

**Pull request merged**